alphadia/workflow/base.py

```diff
@@ -82,9 +83,18 @@ def load(
         self.reporter.context.__enter__()
         self.reporter.log_event("section_start", {"name": "Initialize Workflow"})
 
-        self.reporter.log_event("loading_data", {"progress": 0})
         # load the raw data
-        self._dia_data = self._get_dia_data_object(dia_data_path)
+        self.reporter.log_event("loading_data", {"progress": 0})
+        raw_file_manager = RawFileManager(
+            self.config,
+            path=os.path.join(self.path, self.RAW_FILE_MANAGER_PATH),
+            reporter=self.reporter,
+        )
+
+        self._dia_data = raw_file_manager.get_dia_data_object(dia_data_path)
+        raw_file_manager.calc_stats(self._dia_data)
```

Comment:
What do you think about always saving the stats as part of `raw_file_manager.get_dia_data_object` ?